Script/item.dat

```diff
@@ -1182,6 +1327,40 @@ scrollofteleportation /* scroll-> */
   ReadDifficulty = 50;
 }
 
+scrolloffireballs /* scroll-> */
+{
+  Possibility = 20;
+  Price = 1000;
+  PostFix = "of immolation";
+  AttachedGod = LEGIFER;
+  ReadDifficulty = 1000;
+  CanBeCloned = false;
+  IsPolymorphSpawnable = false;
+  Alias = { 4, "scroll of fireballs", "scroll of fireball", "power", "magic"; }
+}
+
+scrollofearthquake /* scroll-> */
+{
+  Possibility = 10;
+  Price = 1000;
+  PostFix = "of earthquake";
+  AttachedGod = SILVA;
+  ReadDifficulty = 2500;
+  Alias = { 2, "scroll of earthquakes", "earthquake"; }
+}
+
+scrollofbodyswitch /* scroll-> */
+{
+  Possibility = 2;
+  Price = 3000;
+  PostFix = "of body switch";
+  CanBeWished = false;
+  IsPolymorphSpawnable = false;
+  CanBeCloned = false;
+  AttachedGod = INFUSCOR;
```

Comment:
Maybe Scabies?

Reply:
IDK, this feels more like demonic possession/mind manipulation than transformation/mutation, so I'd say Infuscor.

Reply:
Gonna agree with re-kangaroo, Infuscor is the goddess of wrong knowledge and vile magic, and bodysnatching is absolutely all kinds of wrong and very vile.

Reply:
Ok I agree now, thanks guys